### PR TITLE
rest: add get method for logout

### PR DESCRIPTION
**Pull request by kprzerwa**
* fixes redirection after logout from flask-security

### CODE REVIEW

**Comment by ntarocco** [comment]
@slint anything against this?
The issue we have is that we need to instruct the browser to redirect to CERN logout after the Invenio logout, and it is easier to user a link `<a />` (GET request) to the endpoint then via a POST request in the React app.

**Review by slint — APPROVED** [review]
LGTM, and totally fine since just a feature addition :)

Also, this is basically how it's done today for the UI app

File: setup.py [hunk 1]
```diff
@@ -76,7 +75,7 @@
     'future>=0.16.0',
     'invenio-base>=1.2.2',
     'invenio-i18n>=1.2.0',
-    'invenio-celery>=1.1.2',
```
Comment (ntarocco) [inline setup.py:79]: curiosity, was this change needed?

File: setup.py [hunk 2]
```diff
@@ -22,9 +22,8 @@
     'mock>=2.0.0',
     'pydocstyle>=1.0.0',
     'pytest-cov>=1.8.0',
-    'pytest-flask>=0.10.0,<1.0.0',
```
Comment (ntarocco) [inline setup.py:25]: You can remove more deps here, double check `pytest-invenio` deps

File: run-tests.sh [hunk 3]
```diff
@@ -6,9 +6,7 @@
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
-
-pydocstyle invenio_accounts tests docs && \
-isort -rc -c -df && \
+docker-services-cli up && \
```
Comment (ntarocco) [inline run-tests.sh:9]: probably you don't need this. You can have this file as in the module cookiecutter, without `docker-services-cli`:

```
python -m check_manifest --ignore ".travis-*" && \
python -m sphinx.cmd.build -qnNW docs docs/_build/html && \
python -m pytest
python -m sphinx.cmd.build -qnNW -b doctest docs docs/_build/doctest
tests_exit_code=$?
exit "$tests_exit_code"
```

File: setup.py (hunk 1 above)
Comment (kprzerwa) [reply setup.py:79]: 👀 mistake, thanks

**Review by ntarocco — APPROVED** [review]
